index.js

```diff
@@ -46,15 +46,40 @@ class CoverageBase {
     /**
      * Get shell command to upload coverage to server
      * @method getUploadCoverageCmd
-     * @return {Promise}     Shell commands to upload coverage
+     * @param   {Object}  config
+     * @param   {String}  config.build   build configuration
+     * @return {Promise}  Shell commands to upload coverage
      */
-    getUploadCoverageCmd() {
+    getUploadCoverageCmd(config) {
+        if (this.isCoverageEnabled(this.config, config.build) === 'false') {
+            const skipMessage = 'Coverage feature is skipped. ' +
+                'Set COVERAGE_PLUGIN_ENABLED environment true, if you want to get coverages.';
+
+            return Promise.resolve(`echo ${skipMessage}`);
+        }
+
         return this._getUploadCoverageCmd();
     }
 
     _getUploadCoverageCmd() {
         return Promise.reject('Not implemented');
     }
+
+    /**
+     * Verify to run coverage plugin or not
+     * @method isCoverageEnabled
+     * @param   {Object}  clusterConfig  coverage plugin setting at cluster level.
+     * @param   {Object}  buildConfig.environment.COVERAGE_PLUGIN_ENABLED coverage plugin setting in each builds.
```

Comment:
Is this object?

Reply:
[question] Does `buildConfig` has environment variables of a build? I don't know well 😥

Reply:
nvm, It may be https://github.com/screwdriver-cd/models/blob/806ee9c195db16a55eec4da6023c1057402724ef/lib/buildFactory.js#L328-L332

Reply:
yes, that's right.

Reply:
please check the comment :pray:
https://github.com/screwdriver-cd/coverage-base/pull/11/files#r326496027

And do we need `SD_` prefix for users?

Reply:
I fixed comment.
And added `SD_` prefix.